### PR TITLE
feat: update chromatic github action

### PR DESCRIPTION
**Pull request by NatalieLoyzaga**
## Why

- Add chromatic action to run when push event and approved

### CODE REVIEW

File: .github/workflows/chromatic.yaml [hunk 1]
```diff
@@ -8,9 +8,6 @@ on:
   pull_request_review:
     types: [submitted]
   push:
-    paths:
```
Comment (NatalieLoyzaga) [inline .github/workflows/chromatic.yaml:11]: Remove path filter - We want it to trigger for any push and then filter later on

File: .github/workflows/chromatic.yaml [hunk 2]
```diff
@@ -30,13 +26,36 @@ jobs:
       - run: yarn install --frozen-lockfile
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: dorny/paths-filter@v2
```
Comment (NatalieLoyzaga) [inline .github/workflows/chromatic.yaml:29]: Get the changed file paths to be used later

File: .github/workflows/chromatic.yaml [hunk 3]
```diff
@@ -30,13 +26,36 @@ jobs:
...
+          filters: |
+            packages:
+              - 'packages/**'
+            draft:
+              - 'draft-packages/**'
 
       # 👇 Adds Chromatic as a step in the workflow
+
+      # Run when push event and review state is approved
+      - name: Publish to Chromatic - Merge Master into branch
```
Comment (NatalieLoyzaga) [inline .github/workflows/chromatic.yaml:41]: When the PR is approved and merging into master - Run
When the PR is approved and a commit with message that includes "Merge branch 'master' into" - Run

File: .github/workflows/chromatic.yaml [hunk 4]
```diff
@@ -30,13 +26,36 @@ jobs:
...
+        # Chromatic GitHub Action options
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          # 👇 Chromatic projectToken, refer to the manage page to obtain it.
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          autoAcceptChanges: master # 👈 Auto accept master builds (why https://www.chromatic.com/docs/github-actions#github-squashrebase-merge-and-the-main-branch)
+
+      # Run when 'package' or 'draft-packages' files where changes
       - name: Publish to Chromatic
+        if: (steps.changes.outputs.packages == 'true' || steps.changes.outputs.draft == 'true') || (github.event.action == 'labeled' && github.event.label.name == 'run chromatic')
```
Comment (NatalieLoyzaga) [inline .github/workflows/chromatic.yaml:53]: If files changes are within package or draft package - Run
If a label of run chromatic has been added - run

**Review by mcwinter07 — APPROVED** [review]
Looks good to me, really appreciate the comments throughout the yaml file to walk through the new steps. 🤞 it works on first go 😉

**Review by mcwinter07 — DISMISSED** [review]

**Review by mcwinter07 — DISMISSED** [review]

**Review by mcwinter07 — APPROVED** [review]

**Review by mcwinter07 — APPROVED** [review]

**Pull request merged**